Isabelle Unicode for Bitbucket.user.js

```diff
@@ -15,6 +15,9 @@
 
 /*
   CHANGELOG
+  v0.4.5 2023-10-26
+  - fix format for github comment view
+  - add support for github suggested changes
```

Comment:
Is that all? I thought you fixed three things, or do I remember wrong?
nitpick: not sure there's a "comment view", what do you mean? Do you mean the *commit* view?

Reply:
No, should just be those two changes in this version. You might be thinking of something I did in 0.4.4 as part of yesterday's changes.

Comment view is the name that was previously given to the format for the conversation page on Github, i.e. this page we're on right now. See https://github.com/au-ts/isabelle-symbols-bitbucket-github/pull/4/files#diff-3ba464c7dd789065b8dc53c0c0f2e0ecc5fc3db50a05890d5d8b57f784ad78afR506

Reply:
Oh right, I guess it's "discussion view" now, but yours is historically accurate